docs/voice/overview.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Voice in Home Assistant"
+sidebar_label: Overview
+---
+
+:::info
+
+Voice in Home Assistant is a work in progress. Not all parts described on this page have been implemented.
+
+:::
+
+Building a voice assistant is a complex task. It requires a lot of different technologies to work together. This page will give you an overview of the different parts inside Home Assistant and how they will work together.
+
+
+```mermaid
+graph TD;
+    U((User))
+    STT[Speech-to-Text]
+    VA[Voice Assistant]
+    C[Conversation]
+    I[Intent]
+    TTS[Text-to-Speech]
+    U -->|1. Speech + Metadata| VA;
+    VA -->|2. Speech| STT
+    STT -->|3. Sentence| VA
+    VA -->|4. Sentence +\n Metadata| C
+    C -->|5. Intent| I
+    I -->|6. Intent Response| C;
+    C -->|7. Intent Response| VA;
+    VA -->|8. Intent Response| TTS
+    TTS -->|9. Response Audio| VA
+    VA -->|10. Response Audio| U
+```
+
+ - The **Voice Assistant** integration is responsible for turning the user's speech into text, get it processed, and turn the response into speech.
+ - The conversation integration is responsible for processing user's text. It does this by matching it to an intent.
```

Comment:
Why isn't the conversation integration in bold?

Reply:
```suggestion
 - The **conversation** integration is responsible for processing user's text. It does this by matching it to an intent.
```